### PR TITLE
Enable followlinks on os.walk for recursive walk

### PR DESCRIPTION
**Pull request by octavflorescu**
I am loading several ImageDataBunch.from_folder having the same validation set, but experimenting with different training sets.
In order not to have to copy-paste the validation images to each dataset, I have created a symlink (ln -s ../valid ./valid) in each dataset, pointing to the common validation set, but fastai did not use this folder without having set the followlinks param to true.

### CODE REVIEW

**Comment by sgugger** [comment]
Seems a little bit dangerous and it makes the tests fail. Even with the test succeeding, I think this should be an additional param that defaults to `False`.

**Comment by sgugger** [comment]
Thanks! Tests were failing due to some unrelated bug with the release of PyTorch 1.2.0.
